src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -139,7 +145,12 @@ cdef class _AioCall:
             self._destroy_grpc_call()
 
         if receive_status_on_client_operation.code() == StatusCode.ok:
-            return receive_message_operation.message()
+            return UnaryUnaryOpsResult(
+                initial_metadata=receive_initial_metadata_operation.initial_metadata(),
+                message=receive_message_operation.message(),
+                code=receive_status_on_client_operation.code(),
+                details=receive_status_on_client_operation.details(),
+                trailing_metadata=receive_status_on_client_operation.trailing_metadata())
```

Comment:
I would suggest think twice about the `UnaryUnaryOpsResult` design. Under streaming cases, the information is not going to arrive in one batch, and allows it to return just once. Otherwise, there will be two totally different designs of returning information from C-Core to application.

Reply:
In the PR for implementing the interceptors Im gonna segregate these
attributed as attributes belonging to the aio call object. Just for
unblocking the work here, if you agree, I would consider this namedtuple as
a temporary thing.

Reply:
Personally, if this PR is blocking your work in interceptors, I would say they are probably better check in together? The amount of code is not huge, should be fine merging into what you have in your refactor version.

This temporary design here is on critical data path. My intention is that if you are sensing that this needs to be refactored soon, why not perform the refactor all together?

Reply:
It's not really blocking me but if it gets merged first than mine the `namedtuple` would be removed.

But true that the number of changes is not so big, If everybody agrees I can ask @ZHmao for opening the PR against my branch and deliver later a PR which will address both things, hte interceptor and support for metadata.

Do you kinda agree @ZHmao? If so, could you open the PR against this branch [1] and close this?

[1] https://github.com/Skyscanner/grpc/tree/client_unaryunary_interceptors